hopp/utilities/utilities.py

```diff
@@ -25,3 +25,29 @@ def load_yaml(filename, loader=Loader) -> dict:
         return filename  # filename already yaml dict
     with open(filename) as fid:
         return yaml.load(fid, loader)
+
+def check_create_folder(filepath):
+    if not os.path.isdir(filepath):
+        os.makedirs(filepath,exist_ok=True)
+        already_exists = False
+    else:
+        already_exists = True
```

Comment:
This could be simplified by setting the variable as `True` just before the `if` block.